src/spikeinterface/postprocessing/amplitude_scalings.py

```diff
@@ -13,16 +13,24 @@
 
 from ..core.template_tools import get_dense_templates_array, _get_nbefore
 
-# DEBUG = True
-
-
 # TODO extra sparsity and job_kwargs handling
 
 
 class ComputeAmplitudeScalings(AnalyzerExtension):
     """
     Computes the amplitude scalings from a SortingAnalyzer.
 
+    Amplitude scalings are the scaling factor to multipy the
+    the unit template to best match the waveform. Each waveform
+    has an associated amplitude scaling. This is a more robust
+    estimate of the waveform amplitude that other methods e.g. raw voltage.
+
+    When there are collisions, the scaling is the regression
+    of the waveform onto the template, with intercept:
+        scaling * template + intercept = waveform
+    When there are collisions, a multiple-regression is used to fit
+    the waveform to template while taking into account colliding spikes.
```

Comment:
This could probably be simplified a little no? Seems like you're repeating a bit that it is the regression then you say it is a multiple-regression.